android/capacitor/src/main/java/com/getcapacitor/MessageHandler.java

```diff
@@ -30,15 +33,26 @@ public MessageHandler(Bridge bridge, WebView webView) {
   public void postMessage(String jsonStr) {
     try {
       JSObject postData = new JSObject(jsonStr);
-      String callbackId = postData.getString("callbackId");
-      String pluginId = postData.getString("pluginId");
-      String methodName = postData.getString("methodName");
-      JSObject methodData = postData.getJSObject("options");
 
-      Log.d(Bridge.TAG, "To native: " + callbackId + ", pluginId: " + pluginId +
-          ", methodName: " + methodName);
+      String type = postData.getString("type");
 
-      this.callPluginMethod(callbackId, pluginId, methodName, methodData);
+      if (type!=null && type.equals("cordova")) {
+        String callbackId = postData.getString("callbackId");
+        String service = postData.getString("service");
+        String action = postData.getString("action");
+        String actionArgs = postData.getString("actionArgs");
+        Log.d(Bridge.TAG, "To native: " + callbackId + ", service: " + service +
```

Comment:
Maybe indicate this is a Cordova native call? 

Perhaps "To native (cordova):"